Dockerfile

```diff
@@ -1,27 +1,49 @@
-FROM node:16.15.0-alpine as builder
+# Arguments
+ARG NODE_VERSION=lts-alpine
+
+# NOTE: Ensure you set NODE_VERSION Build Argument as follows...
+#
+#  export NODE_VERSION="$(cat .nvmrc)-alpine" \
+#  docker build \
+#    --build-arg NODE_VERSION=$NODE_VERSION \
+#    -t mojaloop/sdk-scheme-adapter:local \
+#    . \
+#
+
+# Build Image
+FROM node:${NODE_VERSION} as builder
+USER root
+
 WORKDIR /opt/app
 
 RUN apk --no-cache add git
-RUN apk add --no-cache -t build-dependencies make gcc g++ python3 libtool libressl-dev openssl-dev autoconf automake \
-    && cd $(npm root -g)/npm \
-    && npm config set unsafe-perm true \
-    && npm install -g node-gyp
+RUN apk add --no-cache -t build-dependencies make gcc g++ python3 libtool openssl-dev autoconf automake bash \
+    && cd $(npm root -g)/npm
 
-COPY package*.json /opt/app/
+COPY package.json package-lock.json* /opt/app/
 
 RUN npm ci
 
-FROM node:16.15.0-alpine
+RUN apk del build-dependencies
+
+COPY src /opt/app/src
+COPY config /opt/app/config
+
+FROM node:${NODE_VERSION}
+
 WORKDIR /opt/app
 
-# Create a non-root user: ml-user
-RUN adduser -D ml-user 
-USER ml-user
+# Create empty log file & link stdout to the application log file
+RUN mkdir ./logs && touch ./logs/combined.log
+# Links combined to stdout
+RUN ln -sf /dev/stdout ./logs/combined.log
```

Comment:
Why is this necessary?

Reply:
Not sure, it's added into every Dockerfile already, I'll investigate.

Reply:
By linking the standard output to the application log file, any output written to stdout by the application will be redirected and appended to the "combined.log" file. This allows us to access and monitor the application's log output.

Reply:
Ok, thank you for the clarification. 
The `central-service-logger` also defaults to writing to that file when using the file transport for logging:
```
https://github.com/mojaloop/central-services-logger/blob/966826cab74713a2ba743fe1b89eb200acfb7c68/config/default.json#L10
```